### PR TITLE
Add '-pn' option to specify port name.

### PR DESCRIPTION
**Pull request by mckuhei**
E.g `sc55emu.exe "-pn:2- Roland SC-D70 MIDI IN"` to open midi devices, If devices not exist, will fallback to `-p` option.

### CODE REVIEW

File: src/mcu.cpp [hunk 1]
```diff
@@ -1354,6 +1358,28 @@ int main(int argc, char *argv[])
...
+    if (pname != NULL) {
+        int portNo = 0;
+        int length = MIDI_GetMidiInDevices(NULL);
+        auto devices = new char[length];
+        length = MIDI_GetMidiInDevices(devices);
+        auto start = devices;
+        auto end = devices + length;
+        while (start != end) {
+            int len = strlen(start);
+            if (!strcmp(start, pname)) {
```
Comment (NicknineTheEagle) [inline src/mcu.cpp:1370]: IMHO, it's better to use case insensitive check here (_stricmp on MSVC, strcasecmp on POSIX).

**Comment by mckuhei** [comment]
ok

File: src/mcu.cpp (hunk 1 above)
Comment (mckuhei) [reply src/mcu.cpp:1370]: ok

**Comment by mckuhei** [comment]
@nukeykt Please close or merge this pull request.

**Comment by mckuhei** [comment]
It's very annoying to use port number open a midi devices, especially if you have a hardware device.

**Comment by UnBeatWaterGH** [comment]
> command line arguments probably will be redundant once gui and config files are added, these are being done and will be in next version.

I don't think command line arguments will be _completely_ useless.

**Pull request closed without merging**